aws/transit_gateway_types.go

```diff
@@ -0,0 +1,94 @@
+package aws
+
+import (
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+)
+
+// TransitGatewaysVpcAttachment - represents all transit gateways vpc attachments
+type TransitGatewaysVpcAttachment struct {
+	Ids []string
+}
+
+// ResourceName - the simple name of the aws resource
+func (tgw TransitGatewaysVpcAttachment) ResourceName() string {
+	return "transit-gateway-attachment"
+}
+
+// MaxBatchSize - Tentative batch size to ensure AWS doesn't throttle
+func (tgw TransitGatewaysVpcAttachment) MaxBatchSize() int {
+	return 200
```

Comment:
Perhaps this should be set as a `const`?

Reply:
I think that No because it's a `Method`, I misunderstood something?

Reply:
It could be a const, like this:
```
const maxBatchSize = 200
...

return maxBatchSize
```

But it's a small NIT, so feel free to ignore.